### PR TITLE
[test/plugin] immigrate test redisson scenario.

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
https://github.com/apache/skywalking/issues/3583

Workload 2

### CODE REVIEW

File: Jenkinsfile-Agent-Test-2 [hunk 1]
```diff
@@ -81,6 +81,12 @@ pipeline {
             parallel {
                 stage('Group1') {
                     stages {
+                        stage('redisson-scenario 3.x (23)') {
+                            steps {
+                                sh 'bash test/plugin/run.sh --build_id=wl2_${BUILD_ID} redisson-scenario'
+                            }
+                        }
+
```
Comment (arugal) [inline Jenkinsfile-Agent-Test-2]: `Test Case Report` miss

**Comment by wu-sheng** [comment]
Take look at #3774, the command will be changed.

File: test/plugin/scenarios/redisson-scenario/bin/startup.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -Dredis.servers=${REDIS_SERVERS} -jar ${agent_opts} "-Dskywalking.agent.service_name=redisson-scenario" ${home}/../libs/redisson-scenario.jar &
```
Comment (arugal) [inline test/plugin/scenarios/redisson-scenario/bin/startup.sh]: Hi, `agent_opts` already includes `skywalking.agent.service_name`,  so you don't need to set it.

**Comment by wu-sheng** [comment]
This isn't fixing the new version bug, right?

**Comment by zhaoyuguang** [comment]
> Take look at #3774, the command will be changed.

Ok, I will improve this pr when 3774 pr merged

**Comment by wu-sheng** [comment]
@zhaoyuguang That PR merged, please update.

**Comment by zhaoyuguang** [comment]
> @zhaoyuguang That PR merged, please update.

OK

File: Jenkinsfile-Agent-Test-2 (hunk 1 above)
Comment (zhaoyuguang) [reply Jenkinsfile-Agent-Test-2]: 3Q  I  fixed

File: test/plugin/scenarios/redisson-scenario/bin/startup.sh (hunk 2 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/redisson-scenario/bin/startup.sh]: 3Q  I had fixed, please recheck

**Pull request merged**